.travis.yml

```diff
@@ -5,15 +5,22 @@ addons:
 
 language: ruby
 
-before_install: rvm rubygems master --force
```

Comment:
@wjordan we needed this line to fix this problem: https://travis-ci.org/oauth-xx/oauth-ruby/jobs/148184710#L190